### PR TITLE
[sql_lab]Disabled run query button if sql query editor is empty

### LINKED ISSUE #2102: [sql-lab] disable run query button if sql editor is empty
screenshot 2017-02-02 18 25 58

### PR DESCRIPTION
**Pull request by lprashant-94**
Fix 
#2102

Disabled run button if sql query is empty.

### CODE REVIEW

File: superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ export default function RunQueryActionButton(props) {
       onClick={() => props.runQuery(true)}
       key="run-async-btn"
       tooltip={t('Run query asynchronously')}
+      disabled={!props.sql.trim()}
```
Comment (mistercrunch) [inline superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx]: I'm confused as to why the linter isn't asking for `sql` to be in `propTypes`. I wanted to make sure it's entered as a `string` and can never be `null` through a defaultProps = ''

**Review by mistercrunch — CHANGES_REQUESTED** [review]

**Comment by lprashant-94** [comment]
I didn't understand your comment, can you please elaborate.

**Comment by lprashant-94** [comment]
I got it, I will add propsType for sql as well, 

Thanks

File: superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx (hunk 1 above)
Comment (lprashant-94) [reply superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx]: I have made changes, please review :+1:

**Pull request merged**